### PR TITLE
Mafia game v2 jp develop

### PR DESCRIPTION
**Pull request by erniker**
He actualizado los modelos de las tablas de la BD
He unido el backend del BackOffice y el backend del front

### CODE REVIEW

File: backend/controllers/question.js [hunk 1]
```diff
@@ -109,8 +109,14 @@ class Question {
         ]
       })
       .then(questions => {
+        //console.log('My questions are: ')
```
Comment (Frzurita) [inline backend/controllers/question.js]: you can remove these comments

File: core-modules/be-core-modules/data-bases/relationalDataBase/models/prod/Answer.js [hunk 2]
```diff
@@ -2,18 +2,21 @@ module.exports = function(sequelize, DataTypes) {
   var modelName = 'answer'
 
   const Answer = sequelize.define(modelName, {
-    label: {
+    response: {
```
Comment (Frzurita) [inline core-modules/be-core-modules/data-bases/relationalDataBase/models/prod/Answer.js]: This name is actually the label of the response. Feel free to keep it but I think label is a better choice

File: core-modules/be-core-modules/data-bases/relationalDataBase/models/prod/Question.js [hunk 3]
```diff
@@ -10,11 +10,15 @@ module.exports = function(sequelize, DataTypes) {
...
+    hierarchy: {
       type: DataTypes.STRING,
       allowNull: false
     },
-    description: {
+    organization: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    decision: {
```
Comment (Frzurita) [inline core-modules/be-core-modules/data-bases/relationalDataBase/models/prod/Question.js]: what do you mean by decision?

File: package.json [hunk 4]
```diff
@@ -4,11 +4,12 @@
   "description": "This is going to be the service to retrieve data from all the different sources we are going to have.",
   "main": "server.js",
   "scripts": {
+    "dev": "nodemon node server.js",
```
Comment (Frzurita) [inline package.json]: you can add the prestart script here
```suggestion
    "dev": "npm run prestart && nodemon node server.js",
```

File: product/routes/index.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+router
+  .get(
+    '/helloworld',
+    controllers.helloworld.helloworld
+  )
+  .get(
+    '/question/list',
+    controllers.question.list
+  )
+  .get(
```
Comment (Frzurita) [inline product/routes/index.js:15]: Nice. Right now you are picking up a random question. We will improve this request a bit more in the next iteration

**Review by Frzurita — COMMENTED** [review]
Good job! I left a few comments so you can check them out. Let me know if you have any question about them. Cheers!

File: backend/controllers/question.js (hunk 1 above)
Comment (erniker) [reply backend/controllers/question.js]: Fixed

File: core-modules/be-core-modules/data-bases/relationalDataBase/models/prod/Answer.js (hunk 2 above)
Comment (erniker) [reply core-modules/be-core-modules/data-bases/relationalDataBase/models/prod/Answer.js]: response is de response that you choise, so, y called it in that way

File: core-modules/be-core-modules/data-bases/relationalDataBase/models/prod/Question.js (hunk 3 above)
Comment (erniker) [reply core-modules/be-core-modules/data-bases/relationalDataBase/models/prod/Question.js]: Is de decision you have to answered by choising a response

File: package.json (hunk 4 above)
Comment (erniker) [reply package.json]: Fixed

File: core-modules/be-core-modules/data-bases/relationalDataBase/models/prod/Question.js (hunk 3 above)
Comment (Frzurita) [reply core-modules/be-core-modules/data-bases/relationalDataBase/models/prod/Question.js]: I don't quite understand it. We can talk about it through a phone call

**Review by Frzurita — APPROVED** [review]

File: core-modules/be-core-modules/data-bases/relationalDataBase/models/prod/Answer.js (hunk 2 above)
Comment (erniker) [reply core-modules/be-core-modules/data-bases/relationalDataBase/models/prod/Answer.js]: Fixed

File: core-modules/be-core-modules/data-bases/relationalDataBase/models/prod/Question.js (hunk 3 above)
Comment (erniker) [reply core-modules/be-core-modules/data-bases/relationalDataBase/models/prod/Question.js]: fixed

**Comment by erniker** [comment]
There are still two problems:
The delete button does not delete when there are related questions
The initial settings panel do nothing
Maybe is front problems

**Pull request merged**